wazimap_ng/config/common.py

```diff
@@ -334,6 +334,8 @@ def get_env_value(env_variable):
 
 STAFF_GROUPS = ["ProfileAdmin", "DataAdmin"]
 
+STAFF_EMAIL_ADDRESS = "openup@org.ca"
```

Comment:
can we make this into an environment variable? thanks.

Reply:
Update PR @milafrerichs 
Thanks for review